### DISCUSSION TITLE
Smartthing e HA

### DISCUSSION THREAD

**Discussion by lucaxxaa** [indomus/forum#1135] [HUB: Home Assistant]
Ciao.
Mi aiutate a configurare integrazione di HA smartthing?
Per configurare webhook, cosa devo inserire in config.yaml
Attualmente uso duckdns su HA
Il token Samsung lo ho, le porte sul router sono già reindirizzate ( 443 , 8123) ma non capisco il fatto che smartthing necessità di un certificato.
Quando accedo alla integrazione ed inserisco il token, e non raggiunge la pagina webhook

l' integrazione genera questo link https://miodominio.duckdns.org/api/webhook/*********************************** che deve essere raggiungibile da internet, ma gia come prova, se ci clicco sopra, non carica la pagina. 
Se invece ci metto io la porta manialmente, nell' indirizzo, lo carica. 

![Screenshot_2021-12-03-22-40-03-77](https://user-images.githubusercontent.com/79480865/144676282-4575dd02-0d9c-48d9-9b0c-48b42fb35e52.jpg)

**Comment by lucaxxaa** [comment]
..risolto, era un problema di configurazione port forward su router

**Reply by roby70cr** [reply]
e quindi come hai risolto?
